controllers/controllers.js

```diff
@@ -69,3 +70,20 @@ exports.postComment = (request, response, next) => {
       next(error);
     });
 };
+
+exports.patchReview = (request, response, next) => {
+  const { review_id } = request.params;
+  const { inc_votes } = request.body;
+
+  const selectReviewsPromise = selectReviewsById(review_id);
+  const updateReviewPromise = updateReview(review_id, inc_votes);
+
+  Promise.all([selectReviewsPromise, updateReviewPromise])
+    .then((result) => {
+      const updatedReview = result[1];
+      response.status(200).send({ updatedReview });
```

Comment:
This should be sent back on the key of `review`, changing the varibale name could cause som problems when you get to front end!